core/src/main/java/com/linecorp/armeria/common/stream/FixedStreamMessage.java

```diff
@@ -50,6 +50,7 @@
     private volatile CloseEvent closeEvent;
 
     private int requested;
+    private volatile boolean cancelled;
```

Comment:
Is this necessary? Can we just check if `closeEvent` is `null`?

Reply:
I'd tried it for the first time.
The `closeEvent` is set to null while `cleanup(...)`. Is it better not to set null if the `closeEvent` is `CANCELLED_CLOSE`? 🤔 

https://github.com/line/armeria/blob/c95c0970d7289aafbf167208c532c4e4ebad6fb0/core/src/main/java/com/linecorp/armeria/common/stream/FixedStreamMessage.java#L64-L66